apps/ehr/src/features/external-labs/components/labs-orders/LabsTable.tsx

```diff
@@ -178,148 +270,201 @@ export const LabsTable = <SearchBy extends LabOrdersSearchBy>({
   };
 
   return (
-    <Paper
-      sx={{
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        gap: 3,
-        mt: 2,
-        p: 3,
-        position: 'relative',
-      }}
-    >
-      {loading && <LabOrderLoading />}
-
-      {titleText && (
-        <Typography
-          variant="h3"
-          color="primary.dark"
-          sx={{ mb: -2, mt: 2, width: '100%', display: 'flex', justifyContent: 'flex-start' }}
+    <>
+      {showSubmitBanner && (
+        <Box
+          sx={{
+            width: '100%',
+            height: '48px',
+            backgroundColor: `${otherColors.lightGreen}`,
+            display: 'flex',
+            justifyContent: 'left',
+            alignItems: 'center',
+            borderRadius: '4px',
+            py: '6px',
+            px: '16px',
+          }}
+          gap="12px"
         >
-          {titleText}
-        </Typography>
+          <CheckCircleOutlineIcon sx={{ color: `${theme.palette.success.main}` }}></CheckCircleOutlineIcon>
+          <Typography variant="button" sx={{ textTransform: 'none', color: `${theme.palette.success.dark}` }}>
+            Tests are ready to be sent. Please review then select submit.
```

Comment:
Just a suggestion to shorten the text
```suggestion
            Tests are ready to be sent. Please review then Submit.
```